### ISSUE TITLE
The inference time of A-Vit same as the Deit.

### ISSUE THREAD

**Issue by dk-liang** [NVlabs/A-ViT#2]
Thanks for this interesting work, and I believe it would be valuable for people in this area. 
 Here, I have some problems. Could the authors provide some explanation?
(1) Why the inference time of A-Vit is same as Deit? According to the paper, the A-VIT is faster than Deit. But I find that the inference time is the same regardless of whether the pre-trained model is loaded or not.

(2) According to the paper, different samples should have different tokens, i.e., the tensor shape should not be the same in the testing phase. So, how to evaluate the validation set under 64 batch size? To the best of my knowledge, it is almost impossible to combine into a batch.

**Comment by hongxuyin** [comment]
Hi yes that snippet is not yet released we will release dynamic zipping, distillation, and base model etc. in coming versions. Thanks for feedback and we will update readme to reflect.

**Comment by YuanYeshang** [comment]
hi，i want to ask that if you can release the dynamic zipping now?

**Comment by xyl-507** [comment]
+1
looking forward the dynamic zipping！
